ocdskingfisherprocess/transform/upgrade_1_0_to_1_1.py

```diff
@@ -0,0 +1,83 @@
+from ocdskingfisherprocess.transform.base import BaseTransform
+from ocdskit.upgrade import upgrade_10_11
+import sqlalchemy as sa
+
+
+class Upgrade10To11Transform(BaseTransform):
+    type = 'UPGRADE-1-0-TO-1-1'
```

Comment:
I might prefer an all-lowercase `type` (easier to type) and to change the other transform's `type` slightly to `compile-releases` (make the transform a predicate, not a noun).

Reply:
>  might prefer an all-lowercase type (easier to type) 

I don't see any case where people should be typing these directly, and in any case it should be easy to correct case.

> and to change the other transform's type slightly to compile-releases (make the transform a predicate, not a noun).

Ahh, I have mixed up language here. I'll change 'COMPILED-RELEASES' to 'COMPILE-RELEASES' (and associated stuff) in another small pull request. Thanks.